packages/kbn-search-api-panels/components/ingestions_panel.tsx

```diff
@@ -0,0 +1,146 @@
+        }
+      ),
+      title: i18n.translate(
+        'searchApiPanels.welcomeBanner.ingestData.alternativeOptions.logstashTitle',
+        {
+          defaultMessage: 'Logstash',
+        }
+      ),
+      links: [
+        {
+          href: docLinks.logstash,
+          label: i18n.translate(
+            'searchApiPanels.welcomeBanner.ingestData.alternativeOptions.logstashDocumentationLabel',
+            {
+              defaultMessage: 'Documentation',
+            }
+          ),
+
+          external: true,
+        },
+        {
+          href: 'https://github.com/elastic/logstash',
+          icon: 'logoGithub',
+          label: i18n.translate(
+            'searchApiPanels.welcomeBanner.ingestData.alternativeOptions.logstashGithubLabel',
+            {
+              defaultMessage: 'logstash',
+            }
+          ),
+        },
+      ],
+    },
+    {
+      description: i18n.translate(
+        'searchApiPanels.welcomeBanner.ingestData.alternativeOptions.beatsDescription',
+        {
+          defaultMessage:
+            'Lightweight, single-purpose data shippers for Elasticsearch. Use Beats to send operational data from your servers.',
+        }
+      ),
+      title: i18n.translate(
+        'searchApiPanels.welcomeBanner.ingestData.alternativeOptions.beatsTitle',
+        {
+          defaultMessage: 'Beats',
+        }
+      ),
+      links: [
+        {
+          href: docLinks.beats,
+          label: i18n.translate(
+            'searchApiPanels.welcomeBanner.ingestData.alternativeOptions.BeatsDocumentationLabel',
+            {
+              defaultMessage: 'Documentation',
+            }
+          ),
+          external: true,
+        },
+        {
+          href: 'https://github.com/elastic/beats',
+          icon: 'logoGithub',
+          label: i18n.translate(
+            'searchApiPanels.welcomeBanner.ingestData.alternativeOptions.BeatsGithubLabel',
+            {
+              defaultMessage: 'Beats',
+            }
+          ),
+        },
+      ],
+    },
+  ];
+  return (
+    <>
+      {additionalIngestionPanel}
+      {panels.map(({ title, description, links }) => (
+        <EuiFlexGroup direction="column" justifyContent="spaceEvenly" gutterSize="s">
+          <EuiFlexItem grow={false}>
+            <EuiTitle size="xxs">
+              <h6>{title}</h6>
+            </EuiTitle>
+            <EuiSpacer size="xs" />
+            <EuiText>
+              <p>{description}</p>
+            </EuiText>
+          </EuiFlexItem>
+          {links && links.length > 0 && (
+            <>
+              <EuiFlexGroup direction="row" justifyContent="flexStart" alignItems="center">
+                {links.map(({ label, href, external, icon }, index) => (
+                  <EuiFlexItem grow={false}>
+                    <EuiLink href={href} key={index} external={external}>
+                      {icon ? (
+                        <EuiButtonEmpty color="primary" iconType={icon} size="s">
```

Comment:
instead of nesting this in the `EuiLink` I think it would be better to render the `EuiButtonEmpty` if you have an icon and a `EuiLink` if you don't.